app/Http/Controllers/SubmissionController.php

```diff
@@ -65,7 +66,7 @@ private function failProcessing(?string $filename, int $outResponseCode, string
         if ($filename !== null) {
             try {
                 Storage::move('inbox/' . $filename, 'failed/' . $filename);
-            } catch (UnableToMoveFile) {
+            } catch (UnableToMoveFile $e) {
```

Comment:
There's no need to define a variable if you're not using it.  Did you intend to replace the log message with a `report()`?